cmd/run.go

```diff
@@ -18,6 +18,35 @@ import (
 var RAIL_PORT = 4411
 
 func (h *Handler) Run(ctx context.Context, req *entity.CommandRequest) error {
+	isEphemeral := false
+	for _, arg := range req.Args {
+		if (arg) == "--ephemeral" {
+			isEphemeral = true
+		}
+	}
+
+	projectId, err := h.cfg.GetProject()
+	if err != nil {
+		return err
+	}
+
+	// Get Current Environment for name
+	environment, err := h.ctrl.GetEnvironment(ctx)
+	if err != nil {
+		return err
+	}
+
+	// Add something to the ephemeral env name
+	if isEphemeral {
+		// Create new environment for this run
+		environment, err = h.ctrl.CreateEnvironment(ctx, &entity.CreateEnvironmentRequest{
```

Comment:
We might want to make these an actual ephemeral environment since it will show differently in the UI and allows you to specify a base environment to "fork" env vars from

Reply:
Will do ephemerals + specify current environment